crates/wasmcloud-host/src/dispatch.rs

```diff
@@ -435,7 +435,7 @@ pub(crate) fn gen_config_invocation(
         WasccEntity::Capability {
             contract_id: contract_id.to_string(),
             id: provider_id.to_string(),
-            binding: binding_name,
+            link_name: link_name,
```

Comment:
Here you can actually leave out the right hand side of the expression, just `link_name,` will work with Rust since the variable and struct field have the same name